### PR TITLE
Adding changelogs for 2.0.0

### PR DESCRIPTION
**Pull request by marcoag**
Adding changelogs for the 2.0.0 release, please review content and modify as required.

### CODE REVIEW

File: packages/json/CHANGELOG.rst [hunk 1]
```diff
@@ -0,0 +1,9 @@
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Changelog for package soss-websocket
```
Comment (marcoag) [inline packages/json/CHANGELOG.rst]:
```suggestion
Changelog for package soss-json
```

File: packages/mock/CHANGELOG.rst [hunk 2]
```diff
@@ -0,0 +1,7 @@
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Changelog for package soss-websocket
```
Comment (marcoag) [inline packages/mock/CHANGELOG.rst]:
```suggestion
Changelog for package soss-mock
```

File: packages/ros2-test/CHANGELOG.rst [hunk 3]
```diff
@@ -0,0 +1,10 @@
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Changelog for package soss-websocket
```
Comment (marcoag) [inline packages/ros2-test/CHANGELOG.rst]:
```suggestion
Changelog for package soss-ros2-test
```

File: packages/ros2/CHANGELOG.rst [hunk 4]
```diff
@@ -0,0 +1,12 @@
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Changelog for package soss-websocket
```
Comment (marcoag) [inline packages/ros2/CHANGELOG.rst]:
```suggestion
Changelog for package soss-ros2
```

File: packages/rosidl/CHANGELOG.rst [hunk 5]
```diff
@@ -0,0 +1,9 @@
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Changelog for package soss-websocket
```
Comment (marcoag) [inline packages/rosidl/CHANGELOG.rst]:
```suggestion
Changelog for package soss-rosidl
```

File: packages/websocket-test/CHANGELOG.rst [hunk 6]
```diff
@@ -0,0 +1,10 @@
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Changelog for package soss-websocket
```
Comment (marcoag) [inline packages/websocket-test/CHANGELOG.rst]:
```suggestion
Changelog for package soss-websocket-test
```

**Comment by richiprosima** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/soss/176/)

**Review by jamoralp — APPROVED** [review]

**Comment by richiprosima** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/soss/178/)

**Pull request merged**